identity-server/templates/src/IdentityServer/Program.cs

```diff
@@ -15,8 +16,26 @@
 {
     var builder = WebApplication.CreateBuilder(args);
 
+    // Set up logging to write regular entries to console, and diagnostics data to a file.
+    // See https://docs.duendesoftware.com/identityserver/diagnostics/data
     builder.Host.UseSerilog((ctx, lc) => lc
-        .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level}] {SourceContext}{NewLine}{Message:lj}{NewLine}{Exception}{NewLine}", formatProvider: CultureInfo.InvariantCulture)
+            .WriteTo.Logger(consoleLogger =>
+            {
+                consoleLogger
+                    .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level}] {SourceContext}{NewLine}{Message:lj}{NewLine}{Exception}{NewLine}", formatProvider: CultureInfo.InvariantCulture)
+                    .Filter.ByExcluding(Matching.FromSource("Duende.IdentityServer.Diagnostics.Summary"));
```

Comment:
We should do this in the other templates too, please.

Reply:
Pushed an additional commit 👍